web/graphql-codegen.types.js

```diff
@@ -10,6 +10,18 @@ const prettier = require("prettier");
 module.exports = {
   schema: "graphql/graphql.schema.json",
   ignoreNoDocuments: true,
+  config: {
+    defaultScalarType: "unknown",
+    scalars: {
+      timestamp: "string",
+      timestamptz: "string",
+      jsonb: "any",
+      json: "any",
+      numeric: "number",
+      _text: "any",
```

Comment:
@andy-t-wang I've noticed that for some DB types autogen is making `any` type, which can be improved with this config. For example `supported_countries` field has `text[]` (or `_text`) in the db, which should have `Array<string>` type in the TS, but autogen is making it `any`. 

I've added this config, but some types I still left as `any` because if I'm trying to type it correctly it invokes TS errors in other parts of the application. Had no time to fix them all.